src/Components/ItemExpanded.js

```diff
@@ -21,14 +23,15 @@ const ItemExpanded = ({ findItem }) => {
                     <ItemExpandedImg className="item-image-large" src={require("../assets/myrlene-numa-SnITZTTeJVE-unsplash.jpg")} alt=""/>
                 </ItemImgDiv>
                 <ItemExpandedDiv>
-                    <CategoryP>{item.category}</CategoryP>
+                    <CategoryP className="category">{item.category}</CategoryP>
                     <TitleAmountDiv>
-                        <H2>{item.name}</H2>
-                        <AmountP>Amount: {item.amount}</AmountP>
+                        <H2 className="craft-name">{item.name}</H2>
+                        <AmountP className="amount">Amount: {item.amount}</AmountP>
                     </TitleAmountDiv>
-                    <DescP>{item.description}</DescP>
-                    {/* <button className="contact-button"><a href="mailto:someone@example.com">Contact Crafter</a></button> */}
-                    <Button name="Contact Crafter" link="" />
+                   {item.user && <AmountP className="crafter-name">{item.user.name}</AmountP>}
```

Comment:
This was such an interesting problem, glad we found a solution. Note to look into why this wasnt working as expected more in the future. Maybe GraphQL stuff?